mountpoint-s3/src/main.rs

```diff
@@ -53,15 +55,28 @@ fn init_tracing_subscriber(is_foreground: bool, log_directory: Option<&Path>) ->
         .format(LOG_FILE_NAME_FORMAT)
         .context("couldn't format log file name")?;
 
+    // log directories and files should not be accessible by other users
+    let mut file_options = OpenOptions::new();
+    file_options.mode(0o640).write(true).create(true);
```

Comment:
Why are we allowing write access to log files?

Reply:
Only the owner would have write access, I think it makes sense, for example, they might want to move or delete the log files.